src/apprt/gtk/Surface.zig

```diff
@@ -717,7 +719,12 @@ pub fn redraw(self: *Surface) void {
 }
 
 /// Close this surface.
-pub fn close(self: *Surface, processActive: bool) void {
+pub fn close(self: *Surface, process_active: bool) void {
+    self.closeWithConfirmation(process_active, .{ .surface = self });
+}
+
+/// Close this surface.
+pub fn closeWithConfirmation(self: *Surface, process_active: bool, target: CloseDialog.Target) void {
```

Comment:
Given this is the surface, is our target not always the surface?

Reply:
Nope! When a tab with just one split is closed it'll call this function with the target set to the tab